src/core/server_status.proto

```diff
@@ -463,12 +448,6 @@ message ServerStatus
   //@@
   ModelControlRequestStats model_control_stats = 9;
 
-  //@@  .. cpp:var:: SharedMemoryControlRequestStats shm_control_stats
-  //@@
-  //@@     Statistics for SharedMemoryControl requests.
-  //@@
-  SharedMemoryControlRequestStats shm_control_stats = 10;
```

Comment:
To avoid breaking existing code just mark this as deprecated.

Reply:
Done